From 90055a9f454f63435a0b1106d4e02db965a18c6f Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Wed, 5 Dec 2012 14:23:05 +0200
Subject: [PATCH 114/277] [PORT FROM MAIN] Telephony: Add suspend/resume and MS class handling

BZ: 37735 44153

In 2G networks which doesn't support simultaneous voice
and data, data will be suspended on voice call and sms
operations.

This patch adds the framework config items needed used by the
Oemtelephony for data state reporting.

Change-Id: I638089caf094b3b6146dc7556d208c9b3cea44f1
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
Reviewed-on: http://android.intel.com:8080/79184
Reviewed-by: Fremaux, Pascal <pascal.fremaux@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Badrappan, Jeevaka <jeevaka.badrappan@intel.com>
Tested-by: Louis, NicolasX <nicolasx.louis@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 core/res/res/values/config.xml  |    7 +++++++
 core/res/res/values/symbols.xml |    3 +++
 2 files changed, 10 insertions(+), 0 deletions(-)

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index 2f173dc..551ad72 100755
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -1029,4 +1029,11 @@
         <item>250</item>
         <item>250</item>
     </integer-array>
+
+    <!-- set to true if oem hooks are supported by framework -->
+    <bool name="config_usage_oem_hooks_supported">false</bool>
+    <!-- set the oemhook data state reporting property -->
+    <string name="oemhook_datastate_property"></string>
+    <!-- set the concurrent data support property -->
+    <string name="oemhook_concurrentdata_property"></string>
 </resources>
diff --git a/core/res/res/values/symbols.xml b/core/res/res/values/symbols.xml
index cdc3180..e0b34db 100644
--- a/core/res/res/values/symbols.xml
+++ b/core/res/res/values/symbols.xml
@@ -253,6 +253,7 @@
   <java-symbol type="bool" name="config_enable_emergency_call_while_sim_locked" />
   <java-symbol type="bool" name="config_enable_puk_unlock_screen" />
   <java-symbol type="bool" name="config_mms_content_disposition_support" />
+  <java-symbol type="bool" name="config_usage_oem_hooks_supported"/>
   <java-symbol type="bool" name="config_showMenuShortcutsWhenKeyboardPresent" />
   <java-symbol type="bool" name="config_sip_wifi_only" />
   <java-symbol type="bool" name="config_sms_capable" />
@@ -625,6 +626,8 @@
   <java-symbol type="string" name="numeric_wday1_md1_wday2_md2" />
   <java-symbol type="string" name="numeric_wday1_mdy1_time1_wday2_mdy2_time2" />
   <java-symbol type="string" name="numeric_wday1_mdy1_wday2_mdy2" />
+  <java-symbol type="string" name="oemhook_datastate_property"/>
+  <java-symbol type="string" name="oemhook_concurrentdata_property"/>
   <java-symbol type="string" name="old_app_action" />
   <java-symbol type="string" name="old_app_description" />
   <java-symbol type="string" name="older" />
-- 
1.7.4.1

